Logger.Tests/FileLoggerTests.cs

```diff
@@ -5,6 +5,21 @@ namespace Logger.Tests
     [TestClass]
     public class FileLoggerTests
     {
-        
+        [TestMethod]
+        public void FileLogger_FilePathIsCreated()
+        {
+            // Arrange
+            LogFactory logFac = new LogFactory();
+
+            // Act
+            //Path for file is subject to change if folder location is moved for the text file.
+            logFac.ConfigureFileLogger("P:/.Net Programming Assignment 2/Logger/test.txt");
```

Comment:
This test will fail - don't hard code file paths. If you want your test to pass, change it to be:
```suggestion
            logFac.ConfigureFileLogger("test.txt");
```
Or change it to use `Path.GetRandomFileName()` and store it in a variable.
Make sure to dispose of the file at the end of your unit test.
Your `StreamWriter` will automatically create a file for you.